example/android/app/src/main/AndroidManifest.xml

```diff
@@ -17,12 +17,9 @@
             <category android:name="android.intent.category.LAUNCHER" />
         </intent-filter>
       </activity>
-      <activity android:name="com.facebook.react.devsupport.DevSettingsActivity" />
-        <meta-data
-            android:name="com.google.android.geo.API_KEY"
-            android:value="AIzaSyDKnc3IIn5VkykUjv5V7u8qBzb4q3OS3tk"/>
+      <activity android:name="com.facebook.react.devsupport.DevSettingsActivity" />        <meta-data
+          android:name="com.google.android.geo.API_KEY"
+          android:value="AIzaSyBAefhRlXEH3vCko-zZTX6PHllTR6av4WI"/>
```

Comment:
not important right now, but we will probably want to remove this api key when publishing the real package